WcaOnRails/config/locales/en.yml

```diff
@@ -1038,20 +1037,19 @@ en:
       committees: "The following committees are responsible for the operational activities of the WCA:"
       #context: WDC
       wdc:
-        name_html: "<strong>WCA Disciplinary Committee (WDC)</strong>"
         description: "This committee advises the WCA Board in special cases, like alleged violations of WCA Regulations, and may be contacted by WCA members in case of important personal matters regarding WCA competitions."
       #context: WRC
       wrc:
-        name_html: "<strong>WCA Regulations Committee (WRC)</strong>"
         description: "This committee is responsible for managing the WCA Regulations."
       #context: WRT
       results:
-        name_html: "<strong>WCA Results team</strong>"
         description: "This team is responsible for managing all competition results."
       #context: WST
       software:
-        name_html: "<strong>WCA Software team</strong>"
```

Comment:
I'm up for letting the opportunity for translators to translate team names, as well as keeping a "universal" reference to the team.
I'd suggest changing the name formatting as follow:
```erb
<strong><%= t '[...].name' %> (<%= team.acronym %>)</strong>
```